spring-ai-core/src/test/java/org/springframework/ai/util/StandardLineEndingTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.ai.util;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.ai.util.StandardLineEnding.CR;
+import static org.springframework.ai.util.StandardLineEnding.CRLF;
+import static org.springframework.ai.util.StandardLineEnding.LF;
+import static org.springframework.ai.util.StandardLineEnding.findByLineSeparator;
+
+/**
+ * @author Kirk Lund
+ */
+class StandardLineEndingTest {
+
+	@Test
+	public void lineSeparator_returnsSlashR_forCR() {

```

Comment:
With JUnit 5, please make test case methods package-private.

NOTE: Not sure if this has been consistently done across the Spring AI project/codebase, if not, then I would suggest this be cleaned up in the Spring AI project in a separate PR.

Reply:
Good idea! Looks like they're all public across most of the tests in Spring AI.